### DISCUSSION TITLE
GitHub Project built-in workflows not working :(

### DISCUSSION THREAD

**Discussion by unaihuete93** [community/community#39648] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hello,

I enabled the following built-in workflows, but my Issues do not get added to the project when created :( 
![image](https://user-images.githubusercontent.com/64772417/203036731-e01ff264-092c-4dda-96c7-1807ff2a1441.png)
The Project is linked:
![image](https://user-images.githubusercontent.com/64772417/203036965-400bebba-3521-4821-995e-0e95ce055405.png)
Am I missing something?

**Comment by unaihuete93** [comment]
Okay, I think I just realized the workflow does not import it automatically to the project, it just defines status "TODO". Should I use GitHub Actions then for importing automatically?

**Reply by lukewar** [reply]
Hi @unaihuete93, we are working on adding this type of workflows to projects. In the meantime please use GitHub Actions: https://github.com/actions/add-to-project

**Reply by RafaPazos** [reply]
Hi @lukewar , since which version do we support workflows in the projects like [here](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-built-in-automations)?

I have a Github enterprise installation that don't support these automations and I would like to confirm them that they have to update. In the meanwhile I will use the action you proposed.

**Comment by ossanna16** [comment]
Hi there @unaihuete93 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
